.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,4 @@
 1. [Read and sign the CLA](https://cla.js.foundation/webpack/webpack.js.org). This needs to be done only once. PRs that haven't signed it won't be accepted.
-2. Make sure your PR complies with [the writer's guide](https://webpack.js.org/writers-guide/).
+2. Make sure your PR complies with [the writer's guide](https://webpack.js.org/development/).
```

Comment:
kk I was thinking about it for a sec but for a sec in terms of 'for PR writers' and blub blub... but yes it's somehow awkward 😛 mom...